src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.WebSearch/Main.cs

```diff
@@ -0,0 +1,232 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO.Abstractions;
+using System.Text;
+using ManagedCommon;
+using Wox.Infrastructure;
+using Wox.Plugin;
+using Wox.Plugin.Logger;
+
+namespace Community.PowerToys.Run.Plugin.WebSearch
+{
+    public class Main : IPlugin, IPluginI18n, IContextMenu, IDisposable
+    {
+        private static readonly IFileSystem FileSystem = new FileSystem();
+        private static readonly IPath Path = FileSystem.Path;
+        private static readonly IFile File = FileSystem.File;
+
+        private bool _disposed;
+
+        public string BrowserIconPath { get; set; }
+
+        public string BrowserPath { get; set; }
+
+        public string DefaultIconPath { get; set; }
+
+        public PluginInitContext Context { get; protected set; }
+
+        public string Name => Properties.Resources.plugin_name;
+
+        public string Description => Properties.Resources.plugin_description;
+
+        public List<ContextMenuResult> LoadContextMenus(Result selectedResult)
+        {
+            return new List<ContextMenuResult>(0);
+        }
+
+        public List<Result> Query(Query query)
+        {
+            var results = new List<Result>();
+
+            if (IsActivationKeyword(query)
```

Comment:
This part os obsolete in this plugin because it's for open default browser only.

Reply:
not sure what you mean. this is to show this:
![image](https://user-images.githubusercontent.com/67976161/145091230-6bf0bb0c-ab8e-4e03-8efb-5e2125a15ba4.png)

Reply:
What happened in the browser then? Opening in search mode or on start page. If it's open on the start page I don't see sny reason to have this feature here.

Reply:
It doesn't open the browser, I changed that in the next commit. this just tells the user